en/docs/references/concepts/authorization/pushed-authorization-requests.md

```diff
@@ -0,0 +1,78 @@
+# Pushed authorization requests (PAR)
+
+Generally, when a user logs in to a web application using a user agent, such as a web browser, an OAuth 2.0 authorization request is initiated from the front-channel. This introduces two major challenges,
+
+- The request data is sent in a URL using a query string. Therefore the data is exposed to malicious users and the authorization server cannot guarantee whether the data has not been tampered by a malicious user.
+
+- Complex authorization requests can sometimes be too large for browsers to process.
+
+PAR defines the `/par` endpoint in an authorization server to elminate these issues. When a user logs in, the back-channel sends the authorization payload directly to the authorization server using the `/par` endpoint.  The `/par` endpoint, in response, returns a `request_uri`.
+
+Consequently, the conventional authorization request takes place in the user agent with a single difference: instead of the request data, the URL contains the `request_uri` which acts as a reference to the authorization payload.
```

Comment:
```suggestion
Consequently, the conventional authorization request takes place in the user agent with a single difference: instead of the request data, the URL contains the `request_uri` and `client_id`, which acts as a reference to the authorization payload.
```